promitor-agent-resource-discovery/values.yaml

```
@@ -12,8 +12,11 @@ image:
   tag:
 
 azureAuthentication:
-  appId:
-  appKey:
+  appId: ""
+  appKey: ""
+  managedIdentityId: ""
+  mode: "servicePrincipal"
+  aadpodidbinding: ""
```

I'd love to use this approach instead:
```suggestion
  mode: "ServicePrincipal"
  identity
    id: ""
    key: ""
    binding
```

---

Sounds good to me